Sources/MessagingPush/MessagingPush.swift

```diff
@@ -51,22 +48,15 @@ public class MessagingPush: ModuleTopLevelObject<MessagingPushInstance>, Messagi
      */
     @discardableResult
     @available(iOSApplicationExtension, unavailable)
-    public static func initialize(
-        configure configureHandler: ((inout MessagingPushConfigOptions) -> Void)? = nil
-    ) -> MessagingPushInstance {
+    public static func initialize(withConfig config: MessagingPushConfigOptions = MessagingPushConfigBuilder().build()) -> MessagingPushInstance {
```

Comment:
I am assuming there is a default value provided because the builder has no required params? 

If that's true, would a code comment be valuable to remember this decision?